osu.Game.Tests/OnlinePlay/MultiplayerPlaylistItemTest.cs

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using Bogus;
+using MessagePack;
+using NUnit.Framework;
+using osu.Game.Online.API;
+using osu.Game.Online.Rooms;
+
+namespace osu.Game.Tests.OnlinePlay
+{
+    [TestFixture]
+    public class MultiplayerPlaylistItemTest
+    {
+        [Test]
+        public void TestCloneMultiplayerPlaylistItem()
+        {
+            var faker = new Faker<MultiplayerPlaylistItem>()
```

Comment:
If we're gonna use these I'd much prefer they had RNG seeded. See: https://github.com/bchavez/Bogus?tab=readme-ov-file#determinism.

Yes it probably shouldn't matter, but I think it's best to set that precedent before copy-pastes happen and it starts actually mattering somewhere.